scripts/ci-e2e.sh

```diff
@@ -44,8 +44,8 @@ export IMAGE_OS=${IMAGE_OS}
 export FORCE_REPO_UPDATE="false"
 EOF
 if [[ ${GINKGO_FOCUS:-} == "features" ]]; then
-    mkdir -p "$CAPI_CONFIG_FOLDER"
-    echo "enableBMHNameBasedPreallocation: true" >"$CAPI_CONFIG_FOLDER/clusterctl.yaml"
+  mkdir -p "$CAPI_CONFIG_FOLDER"
+  echo "enableBMHNameBasedPreallocation: true" >"$CAPI_CONFIG_FOLDER/clusterctl.yaml"
```

Comment:
This has been changed by the following PR: https://github.com/metal3-io/cluster-api-provider-metal3/pull/1751/files

Reply:
That is only on the main branch, we are not backporting it.

Reply:
Thanks got it